example/config/crd/bases/helm.sdk.operatorframework.io_nginxes.yaml

```diff
@@ -16,31 +13,12 @@ spec:
     singular: nginx
   scope: Namespaced
   versions:
-  - name: v1
-    schema:
-      openAPIV3Schema:
-        description: Nginx is the Schema for the nginxes API
-        properties:
-          apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
-              of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
-            type: string
-          kind:
-            description: 'Kind is a string value representing the REST resource this
-              object represents. Servers may infer this from the endpoint the client
-              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
-            type: string
-          metadata:
-            type: object
-          spec:
-            description: NginxSpec defines the desired state of Nginx
-            type: object
-          status:
-            description: NginxStatus defines the observed state of Nginx
-            type: object
-        type: object
-    served: true
-    storage: true
-    subresources:
-      status: {}
+    - name: v1
+      schema:
+        openAPIV3Schema:
+          type: object
+          x-kubernetes-preserve-unknown-fields: true
```

Comment:
We can actually be a bit better than we have been before. This line (`x-kubernetes-preserve-unknown-fields: true`) can be appended to the existing `spec` and `status` properties, like this:

```yaml
spec:
  description: NginxSpec defines the desired state of Nginx
  type: object
  x-kubernetes-preserve-unknown-fields: true
status:
  description: NginxStatus defines the observed state of Nginx
  type: object
  x-kubernetes-preserve-unknown-fields: true
```

Reply:
I will check but I would not too much worried about the CRD. 
Will not it be gen in the next step with the plugin? 
IMO the goal here now is: make the example works for we are able to check helm-operator and figure out what files/customizations should or not be done.